composer.json

```diff
@@ -1,16 +1,15 @@
 {
-    "name": "dionera/laravel-beanstalkd-ui",
+    "name": "lanffy/laravel-beanstalkd-ui",
     "description": "An Admin UI for Beanstalkd and Laravel",
     "license": "MIT",
     "authors": [
         {
-            "name": "Kai Sassnowski",
-            "email": "ksassnowski@gmail.com"
+            "name": "Lanffy",
+            "email": "rlanffy@gmail.com"
```

Comment:
Please don't change the author. You will be listed as a contributor in the repository instead.